src/dc_location.rs

```diff
@@ -13,7 +13,7 @@ use crate::types::*;
 use crate::x::*;
 
 // location handling
-#[derive(Copy, Clone)]
+#[derive(Clone, Default)]
 #[repr(C)]
```

Comment:
Please remove `repr[C]` please, as this is now has types that are not really usable from C-land anymore.

Reply:
I thought it would make possible to use `calloc` when allocation `dc_kml_t` to initialize `Option` to `None`, but it seems there is no such guarantee. I am going to refactor `dc_kml_t` allocations too to make sure.